dot/rpc/modules/sync_state_integration_test.go

```diff
@@ -19,7 +19,7 @@ import (
 func TestSyncStateModule(t *testing.T) {
 	fp := utils.GetWestendDevRawGenesisPath(t)
 
-	data, err := ioutil.ReadFile(filepath.Clean(fp))
+	data, err := os.ReadFile(filepath.Clean(fp))
 	require.NoError(t, err)
 
```

Comment:
why?

Reply:
Because ioutil is deprecated since go 1.17